skrf/time.py

```diff
@@ -225,7 +225,22 @@ def detect_span(ntwk: 'Network', t_unit: str = "") -> float:
 
     return span / time_lookup_dict[t_unit]
 
+def get_window(window: Union[str, tuple, Callable], Nx: int, **kwargs) -> npy.ndarray:
+    """Calls a custom window function or `scipy.signal.get_window()` depending on the window argument.
 
+    Args:
+        window (Union[str, tuple, Callable]): The callable window function or a valid window string
+            for `scipy.signal.get_window()`.
+        Nx (int): Number of samples
+
+    Returns:
+        npy.ndarray: Window samples
+    """
```

Comment:
switch to Numpy style docstring:
```suggestion
    Parameters
    ----------
    window : str, tuple or Callable
        The callable window function or a valid window string
        for `scipy.signal.get_window()`.
    Nx : int
        Number of samples

    Returns
    -------
    window : npy.ndarray
        Window samples.
    """
```